src/richText.js

```diff
@@ -424,12 +422,12 @@ define([
      */
     function makeBubble(form, xpath) {
         function _parseXPath(xpath, form) {
-            if (CASE_REF_REGEX.test(xpath)) {
+            if (!FORM_REF_REGEX.test(xpath)) {
```

Comment:
I think this means that #user and #case will be noted as the with the same classes and icons. Guessing that that was intentional, but curious what were the arguments against using a different UI for them?